R/data-objects.R

```diff
@@ -144,6 +236,70 @@ iget <- function(
   # expand logical path to absolute logical path
   logical_path <- get_absolute_lpath(logical_path)
```

Comment:
This way `path` takes a directory and the destination name is necessarily the same as in iRODS, but that doesn't need to be the case, right?
I would rename `path` to `local_path` and change its default value to `basename(logical_path)`, leaving it to the user to specify a different local path if they so wish (even renaming the file). With the following idea:

- `iget('iris.csv')` -> copies 'iris.csv' from iRODS to the local working directory
- `iget('subcollection/iris.csv') -> copies 'iris.csv' from the 'subcollection' collection in iRODS to the local working directory (as 'iris.csv')
- `iget('iris.csv', 'iris_dataset.csv')` -> copies 'iris.csv' from iRODS to the local working directory while renaming it 'iris_dataset.csv'
- `iget('iris.csv', '/home/mariana/somedatasets/iris.csv')` copies 'iris.csv' from iRODS to some absolute local path.

Reply:
all four given examples sound correct to me.

Reply:
Seems like a smart solution. I will implement it.